qiita_pet/webserver.py

```diff
@@ -239,6 +239,7 @@ def __init__(self):
 
         # 404 PAGE MUST BE LAST IN THIS LIST!
         handlers.append((r".*", NoPageHandler))
+        handlers.append((r"/.*", NoPageHandler))
```

Comment:
This line is new, not sure what's supposed to be for. Can you add a comment please?

Reply:
right, this was a debugging line, removing ....